.github/workflows/jit.yml

```diff
@@ -4,10 +4,16 @@ on:
     paths:
       - '**jit**'
       - 'Python/bytecodes.c'
+      - 'Python/optimizer.c'
+      - 'Python/optimizer_analysis.c'
```

Comment:
I think that we can simplify these two into just `Python/optimizer*.c`
Docs: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet